### PR TITLE
Adding League of Legends Logo

### LINKED ISSUE #9262: League of Legends icon
### Brand Name

League of Legends

### Website

https://www.leagueoflegends.com/

### Popularity Metric

The Similarweb rank is 3,459.
https://www.similarweb.com/website/leagueoflegends.com/

### Official Resources for Icon and Color

Logo and icon official assets packete: 
https://brand.riotgames.com/en-us/league-of-legends/logos/ (`LoL_Icon_Flat_Gold.zip` at the end of the site)

SVG flat gold icon:
LoL_Icon_Flat_GOLD

Gold color: `#bf913b`

### Additional Comments

The brand is owned by Riot Games. You already have [their icon](https://github.com/simple-icons/simple-icons/pull/2794) and the next game of [Valorant](https://github.com/simple-icons/simple-icons/pull/7398) here.

### Contributing

No

### PR DESCRIPTION
**Pull request by CodeWithShipraJain**
**Issue:** closes #9262

**Similarweb rank:** 3,459. See https://www.similarweb.com/website/leagueoflegends.com/
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6927,6 +6927,11 @@
             "hex": "199900",
             "source": "https://github.com/Leaflet/Leaflet/blob/d843c3b88486713827d7e860b58bdba75bfbd5a2/src/images/logo.svg"
         },
+        {
+            "title": "League of Legends",
+            "hex": "BF913B",
```
Comment (adamrusted) [inline _data/simple-icons.json:6932]:
```suggestion
            "hex": "C28F2C",
```

Taken from the source icon on the homepage.

File: icons/leagueoflegends.svg
Comment (adamrusted) [inline icons/leagueoflegends.svg:1]: The monochrome icon on the homepage contains an inner circle too. We should follow their precedent.
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]
Looks perfect, thanks @CodeWithShipraJain 🎉

**Comment by LitoMore** [comment]
Great job @CodeWithShipraJain! I love this one!

**Pull request merged**